dev/DynamicDependency/WinRTPackage.cpp

```diff
@@ -180,17 +180,27 @@ void MddCore::WinRTPackage::ParseAppxManifest_InProcessServer(
                     }
 
                     // Next!
-                    hr =xmlReader->MoveToNextAttribute();
+                    hr = xmlReader->MoveToNextAttribute();
                 }
                 THROW_HR_IF_MSG(APPX_E_INVALID_MANIFEST, activatableClassId.empty() || (threadingModel == MddCore::WinRT::ThreadingModel::Unknown), "Error 0x%X parsing <ActivatableClass> in %ls", hr, filename.c_str());
                 winrtInProcModule.AddInprocServer(activatableClassId, threadingModel);
             }
         }
+        else if (nodeType == XmlNodeType_EndElement)
+        {
+            PCWSTR localName{};
+            THROW_IF_FAILED(xmlReader->GetLocalName(&localName, nullptr));
+            if (CompareStringOrdinal(localName, -1, L"InProcessServer", -1, FALSE) == CSTR_EQUAL)
+            {
+                // Found </InProcessServer> so we're done
+                break;
+            }
+        }
     }
     THROW_HR_IF_MSG(APPX_E_INVALID_MANIFEST, winrtInProcModule.Path().empty() || winrtInProcModule.InprocServers().empty(), "Error 0x%X parsing <ActivatableClass> in %ls", APPX_E_INVALID_MANIFEST, filename.c_str());
 
     // Gotcha!
-    m_inprocModules.push_back(std::move(winrtInProcModule));
+    AddInprocModule(winrtInProcModule);
```

Comment:
You should probably still `move` here and use `&&` syntax on the method, since it might move-from `winrtInProcModule` leaves this local here in the moved-from state.

Reply:
Good idea. Looking to cook this quickly for others' testing so I'll add do that to a future PR.